### DISCUSSION TITLE
modifing some html, js file

### DISCUSSION THREAD

**Discussion by sonagi784** [cvat-ai/cvat#3208] [General]
in web page, I want to change some words but nothing changed

for example, in cvat/cvat-ui/src/index.html
I changed  `<title>Computer Vision Annotation Tool</title>`    to `<title>test</title>`
but nothing changed in website.

restarting docker and using chrome secret brower are useless.

**Comment by azhavoro** [comment]
@sonagi784 Hi, you need to rebuild the images with `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d --build`

**Reply by sonagi784** [reply]
thank you. everything goes well

**Reply by vishalims095** [reply]
thank you it's working fine
